app/javascript/redux/greetingsSlice.js

```diff
@@ -0,0 +1,55 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// export const getRandomGreeting = createAsyncThunk('greetings/fetch', () => (
+//   new Promise((resolve, reject) => {
+//     const options = {
+//       method: 'GET',
+//       url: 'http://localhost:3000/api/greeting',
+//     };
+//     axios.request(options)
+//       .then(({ data }) => {
+//         resolve(data);
+//       })
+//       .catch((error) => {
+//         reject(error);
+//       });
+//   })
+// ));
```

Comment:
- [ ] Your code is great all functionalities working, but this is an issue that is blocking me to make your code approve. Pleae remove them :)